From 058b6757a291086553f2aa467ffbf0bdce102d73 Mon Sep 17 00:00:00 2001
From: Michael Achenbach <machenbach@chromium.org>
Date: Fri, 23 Jul 2021 09:56:56 +0200
Subject: [PATCH] Fix OWNERS for port files

No-Try: true
Bug: chromium:1231999
Change-Id: I08cba762a7ef28cfa2ef74a23e86ac3d057db8dd
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3046188
Reviewed-by: Liviu Rau <liviurau@chromium.org>
Reviewed-by: Hannes Payer <hpayer@chromium.org>
Commit-Queue: Michael Achenbach <machenbach@chromium.org>
Cr-Commit-Position: refs/heads/master@{#75876}
---
 OWNERS | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/OWNERS b/OWNERS
index 6f0ac017208..f9b23f237f5 100644
--- a/OWNERS
+++ b/OWNERS
@@ -27,8 +27,8 @@ per-file codereview.settings=file:INFRA_OWNERS
 per-file AUTHORS=file:COMMON_OWNERS
 per-file WATCHLISTS=file:COMMON_OWNERS
 
-per-file *-mips*=file:MIPS_OWNERS
-per-file *-mips64*=file:MIPS_OWNERS
-per-file *-ppc*=file:PPC_OWNERS
-per-file *-riscv64*=file:RISCV_OWNERS
-per-file *-s390*=file:S390_OWNERS
+per-file ...-mips*=file:MIPS_OWNERS
+per-file ...-mips64*=file:MIPS_OWNERS
+per-file ...-ppc*=file:PPC_OWNERS
+per-file ...-riscv64*=file:RISCV_OWNERS
+per-file ...-s390*=file:S390_OWNERS
-- 
2.35.1

